packages/ui-pagination/src/Pagination/index.tsx

```diff
@@ -402,8 +398,7 @@ class Pagination extends Component<PaginationProps> {
     if (!this.props.children) return null
 
     const currentPageIndex = fastFindIndex(
-      this.props.children,
-      // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'p' implicitly has an 'any' type.
+      this.childPages,
```

Comment:
is this a bugfix?

Reply:
Technically not, it was working before too, it is just typed better this way.